lib/routes/account.js

```diff
@@ -447,15 +518,22 @@ module.exports = function (
           }
         }
 
-        function sendNewDeviceLoginNotification () {
-          if (config.newLoginNotificationEnabled && wantsKeys(request)) {
-            // The response doesn't have to wait for this,
-            // so we don't return the promise.
-            mailer.sendNewDeviceLoginNotification(
-              emailRecord.email,
```

Comment:
If the user does not get the new sign-in confirmation experience, I think we should continue sending them the "new device login" email.  Otherwise it will be a downgrade of security for those users.